### PR TITLE
feat: split incidents UI

### LINKED ISSUE #2461: [➕ Feature]: split incidents UI
- [ ] checkboxes and group actions on `/incidents/{id}/alerts` page
- [ ] modal confirming the change 
- [ ] (?) refactor alerts-table so it could be re-used
- [x] #2875

### PR DESCRIPTION
**Pull request by Kiryous**
Closes #2461 

## Demo
[<br/>Watch Loom](https://www.loom.com/share/9fe472006faf4a899f616d1696784270)

### CODE REVIEW

File: keep-ui/components/table/Pagination.tsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 import { Button, Text } from "@tremor/react";
-import { StylesConfig, SingleValueProps, components, GroupBase } from 'react-select';
-import Select from 'react-select';
+import {
+  StylesConfig,
+  SingleValueProps,
+  components,
+  GroupBase,
+} from "react-select";
+import Select from "react-select";
```
Comment (shahargl) [inline keep-ui/components/table/Pagination.tsx:15]: why here you don't use `Select` from `shared/ui`?

File: keep-ui/shared/ui/Select/Select.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
+"use client";
```
Comment (shahargl) [inline keep-ui/shared/ui/Select/Select.tsx:1]: how isMulti is used here with multiselect? isn't it only the style?

**Review by shahargl — CHANGES_REQUESTED** [review]
two minor comments

File: keep-ui/shared/ui/Select/Select.tsx (hunk 2 above)
Comment (Kiryous) [reply keep-ui/shared/ui/Select/Select.tsx:1]: `isMulti` used for style type annotations here, yes. the prop `isMulti` passed down to react-select

**Review by shahargl — APPROVED** [review]
lgtm

**Pull request merged**